### ISSUE TITLE
Product Images stores full image URL

### ISSUE THREAD

**Issue by JimMackin** [salesagility/SuiteCRM#623]
The product image field in AOS Products stores the full URL of the image. I.e:

```
http://example.com/path/to/suite/upload/someImage.php
```

This isn't ideal since if the SuiteCRM path, domain or protocol (i.e. if they switch to/from https) then the link will be broken. Ideally this should just contain the path relative to the upload directory.

Related to #164

**Comment by recci** [comment]
Note the .htaccess file currently restricts access to the upload directory. You have to remove this restriction for the image to show.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Product Images stores full image URL](https://trello.com/c/uImKLu65/140-product-images-stores-full-image-url)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
